src/Spout/Reader/ODS/RowIterator.php

```diff
@@ -99,15 +108,34 @@ public function valid()
      */
     public function next()
     {
+        $prevRow = null;
+
+        if (count($this->rowDataBuffer) > 1) {
```

Comment:
Can you quickly explain the logic for this change? I'm having trouble understanding how the prevRow/rowDataBuffer work together